mage/test.go

```diff
@@ -1,62 +1,15 @@
-// Copyright (C) 2025 Intel Corporation
 // SPDX-FileCopyrightText: 2025 Intel Corporation
-//
 // SPDX-License-Identifier: Apache-2.0
 
 package mage
 
 import (
-	"strings"
-
 	"github.com/magefile/mage/sh"
 )
 
 func (Test) golang() error {
```

Comment:
should we just remove this if no one is invoking `mage test:golang`?

Reply:
done in [42a680e](https://github.com/open-edge-platform/orch-utils/pull/151/commits/42a680e37da265d2694ae8fd4fedac222a814cd2)